### PR TITLE
Adham week01 homework

### PR DESCRIPTION
**Pull request by Adham-Elias-Botours**
Prettier format 2

### CODE REVIEW

**Comment by Adham-Elias-Botours** [comment]
As I have notified on Slack, it was not possible to submit my homework on Thursday. Now I am submitting it on Friday .
I reused one of my old HTML/CSS homework to make the page of this homework RWD 

also lots of time trying to fix Travis ERROR (Friday)

File: Week1/homework/style.css [hunk 1]
```diff
@@ -1 +1,141 @@
...
+/*Styling step 3: making the style and layout complete 
+depending on media queries*/
+
+
+/*For mobile phones: 2 columns layout.*/
+
+@media only screen and (max-width: 600px) {
+    /*To make a kind of row and 
+  divide the place in it equally on 2 elements*/
+    .books_container>.single_book_item {
```
Comment (nickmcmillan) [inline Week1/homework/style.css]: Do you need the `.books_container>` selector? It's used a couple times in this file. Will it work without it? I'd suggest avoiding nesting CSS selectors if they're not absolutely necessary.

File: Week1/homework/app.js [hunk 2]
```diff
@@ -1,11 +1,140 @@
...
+  booksTitles.forEach(element => {
+    const li = document.createElement('li');
+    ul.appendChild(li);
+    const titleNoUnderScore = element.replace(/_/g, ' ');
+    const titleFirstCapital =
+      titleNoUnderScore.charAt(0).toUpperCase() + titleNoUnderScore.slice(1);
+    li.textContent = titleFirstCapital;
+  });
+};
+const ListBooksTitles = () => {
```
[the remark this answers is not in the document]
Comment (Adham-Elias-Botours) [reply Week1/homework/app.js]: injectListBooksTitles : was the first in use. Then  I copied ListBooksTitles from it, without changing the case.
That is my mistake.

File: Week1/homework/style.css (hunk 1 above)
Comment (Adham-Elias-Botours) [reply Week1/homework/style.css]: I had that feeling when I did that, but I decided to use it with a such single page. I consider your suggestion, and I should avoid that technic.

File: Week1/homework/app.js [hunk 3]
```diff
@@ -1,11 +1,140 @@
...
+    const bookWriter = booksFullInfo[element].writer;
+    const bookCategory = booksFullInfo[element].category;
+    const bookPrice = booksFullInfo[element].price;
+    const bookCover = booksCovers[element];
+    li.innerHTML = `<h3>${bookTitle}</h3><img src="${bookCover}" alt="cover photo of the book: ${bookTitle}"> \
+         <p><span>By: </span>${bookWriter}.<br /><span>Category: </span>${bookCategory}<br /><span>Price:</span> $ ${bookPrice}</p> `;
+  });
+};
+window.addEventListener('load', injectListBooksFullInfo);
+window.addEventListener('load', injectListBooksTitles);
```
[the remark this answers is not in the document]
Comment (Adham-Elias-Botours) [reply Week1/homework/app.js]: thanks, it sounds much logical with just one event listener.

**Comment by Adham-Elias-Botours** [comment]
Thanks for your time Nick. Depending on your advices, I can upgrade this work and then add it to my archive.
Best regards

**Pull request closed without merging**